src/components/IndividualArticleCard.jsx

```diff
@@ -1,4 +1,14 @@
+import { useState } from 'react';
+import ViewComments from './ViewComments';
+
 const IndividualArticleCard = ({ article }) => {
+  const [showComments, setShowComments] = useState(false);
+  const [buttonLabel, setButtonLabel] = useState(true);
```

Comment:
I think this could have a better variable name, it's a bit vague atm unless you know exactly what is happening. revealButton or something along those times would make more sense.